app/utils/Navigator.scala

```diff
@@ -236,9 +236,10 @@ class Navigator @Inject()() extends Logging with FeatureSwitching {
       onSuccessPage = VATRegistrationExceptionId,
       onFailPage = VoluntaryRegistrationId
     ),
-    toNextPage(
+    nextOn(true,
       fromPage = VATRegistrationExceptionId,
-      toPage = TurnoverEstimateId
+      onSuccessPage = ApplyInWritingId,
```

Comment:
should be EligibilityDropoutId(VATExceptionKickoutId.toString) on success to match the other exception kickout

Reply:
After making this change, the registration-exception page now redirects straight to the govuk page - before, it used to send you to the Apply in Writing kickout page on our service. Is this what we want?

Reply:
Also the only way to fix the integration test for this following the change was to hard-code the url into it, since the route it is hitting doesn't exist on our service.